index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Course-R Droid</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="body">
+        <div class="socials">
+            <li><a href=""><img src="images/facebook.png" alt="facebook"></a></li>
+            <li><a href=""><img src="images/twitter.png" alt="twitter"></a></li>
+            <li><a href="">English</a></li>
+            <li><a href="">My page</a></li>
+            <li><a href="">Log Out</a></li>
+        </div>
+    <header>
+        <nav class="mobile-nav">
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#my-courses">Courses</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+            <a href=""><img src="images/x-menu.png" alt="logo" width="20px" height="20px"></a>
+        </nav>
+        <nav class="desktop-nav">
+            <a href="#"><img src="images/logo.png" alt="x-menu" width="50px" height="50px"></a>
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#my-courses">Courses</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+            <div class="mobile-menu-icon">
+              <a href="#">
+                <img src="images/menu.svg" alt="logo" width="30px" height="30px">
+              </a>
+            </div>
+        </nav>
+    </header>
+    <section class="welcome-page">
+        <h1>Welcome to Course-R Droid, where learning becomes fun!</h1>
+        <h2>We are inspired everyday to keep doing what we do when we see creative minds like yours tackling every problem we provide.</h2>
+        <h3>2022.1.10(MON) ~ 2022.6.27(FRI)
+            @231A City Road Wellington, Freetown, Sierra Leone.
+        </h3>
+        <a href=""><div class="button">Apply here</div></a>
+    </section>
+</div>
+    <section class="courses" id="my-courses">
+
+    </section>
+    <div class="senior-tutors">Senior Tutors</div>
+    <section class="tutors">
+        
+    </section>
+    <footer class="partners">
+        <h4>Partners</h4>
+        <hr>
+        <ul>
+            <li><a href="#"><img src="images/adidas.png" alt="adidas"></a></li>
+            <li><a href="#"><img src="images/fedex.png" alt="fedex"></a></li>
+            <li><a href="#"><img src="images/oracle.png" alt="oracle"></a></li>
+        </ul>
+    </footer>
```

Comment:
Good job adding your partners but they are not supposed to be added as a footer please go through the project requirements properly the footer is supposed to be added after the partners sections so the partners are not suppose to be added as a footer.

Please follow these requirements for the footer design:

##### Footer

1. It contains a logo aligned to the left and text aligned to the right.

###### Mobile Size

1. It has a white background.

###### Desktop Size

1. About Page: It has a dark background.
1. Home Page: It has a white background.

Not that this change should be implemented in your about page:

This is your design:
![home 4](https://user-images.githubusercontent.com/79506105/196386965-560ff28f-5ab0-413b-b7ec-8f7888b72546.png)

Please your design should look like this: 
![Home Page 4 Partners and Footer ](https://user-images.githubusercontent.com/79506105/196387071-bc3c192e-f65d-4b56-9045-ee260a904c6d.png)

Please pay attention to detail